src/components/CommunityEvents/CommunityEvents.module.css

```diff
@@ -0,0 +1,87 @@
+.wrapper {
+  margin: 86px 0;
+}
+.component {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: auto;
+  grid-auto-columns: 1fr;
+  align-items: center;
+  gap: 33px;
+  width: 100%;
+}
+
+.contentLeft {
+  justify-self: end;
+  width: 100%;
+  max-width: calc(var(--fullwidth-max-width) / 2);
+  padding-left: var(--space-10x);
+}
+
+.contentRight {
+  display: flex;
+  align-items: center;
+  overflow: hidden;
+}
+
+.sectionHeader {
+  display: flex;
+  gap: var(--space-4x);
+  align-items: end;
+  max-width: var(--fullwidth-max-width);
+  margin-bottom: var(--space-8x);
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0 var(--space-10x);
+}
+
+.arrow {
+  padding: 10px;
+  border: 1px solid var(--border);
+  height: fit-content;
+  cursor: pointer;
+  transition: border-color 0.2s ease;
+}
+
+.arrow:hover {
+  border: 1px solid var(--foreground);
+}
+
+.eventList {
+  display: flex;
+  flex-direction: column;
+  gap: var(--space-6x);
+}
+
+/* Tablet */
+@media (max-width: 1024px) {
+  .component {
+    grid-template-columns: 1fr;
+  }
+
+  .contentRight {
+    margin-left: 0;
+    width: 100%;
+    padding: 0 var(--space-10x);
+  }
+
+  .contentLeft {
+    max-width: 100%;
+  }
+}
+
+/* Mobile */
+@media (max-width: 768px) {
+  .sectionHeader {
+    margin-bottom: var(--space-6x);
+    margin-left: unset;
+    margin-right: unset;
+    & > h2 {
+      font-size: 28px;
+    }
+  }
+
+  .wrapper {
+    margin: 36px 0;
+  }
+}
```

Comment:
On tablet and mobile, the scrolling images should go between the heading and the events. At the moment the images are below the event list